mobile/cmp/toolbar/Toolbar.scss

```diff
@@ -11,7 +11,7 @@
     white-space: nowrap;
     flex-shrink: 0;
 
-    &:not(:first-child):not([class*='xh-no-pad']) {
+    &:not(:first-child):not([class*='xh-no-pad']):not([class*='xh-mask']) {
```

Comment:
What does this change do?

Reply:
Since we moved the appMenu from AppContainer to AppBar, the menu is now a descendant of the toolbar. This means it was getting a left margin by default, which is obviously not desirable on the mask.

Before this change:
<img width="495" alt="image" src="https://user-images.githubusercontent.com/48924151/73020540-d4842a00-3df3-11ea-9eb9-b44e69106cbc.png">
After this change:
<img width="495" alt="image" src="https://user-images.githubusercontent.com/48924151/73020595-f41b5280-3df3-11ea-8fc8-6b832cdcd849.png">

Reply:
Ah great - thanks!